app/components/nav_bar/user_component.html.erb

```diff
@@ -18,17 +18,22 @@
                 <% links.each do |link| %>
                   <%= helpers.link_to_active link.title, link.path, active_class: "bg-gray-900 text-white", class: "text-gray-300 hover:bg-gray-700 hover:text-white px-3 py-2 rounded-md text-sm font-medium" %>
                 <% end %>
-
```

Comment:
I think this comment slipped through the cracks. Can you please add these blank lines back in again?

Reply:
spaces restored in bbec47c455f54437baa20c3e7b1807d326428b9c